auditable-logging-app/Makefile

```diff
@@ -0,0 +1,20 @@
+SHELL := /bin/bash
+
+.PHONY: help
+.DEFAULT_GOAL := help
+
+help: ## 💬 This help message :)
```

Comment:
First time I see emojis in Makefiles ^_^

Reply:
@ross-p-smith originally added it to banking-app. I like it :)